src/cleanfigure.m

```diff
@@ -322,6 +345,152 @@ function pruneOutsideBox(meta, handle)
         0.0 < lambda(:, 2) & lambda(:, 2) < 1.0;
 end
 % =========================================================================
+function [xData, yData] = getVisibleData(meta, handle)
+    % Check whether this is a 3D plot
```

Comment:
[H1 is the MATLAB term for a first line of function help](http://nl.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html?refresh=true). 

For public functions, this should be a short description (=actual H1 line) of what the function does, e.g. 
```matlab
% GETVISIBLEDATA retrieves the data in the visible coordinate frame
```
Such a H1 line makes sure that `lookfor ...` can actually provide useful functions related to `...` (have `...` in their H1 line).

For private functions (such as this one), IMHO we don't have to adhere to H1 comments, as those are not searched by `lookfor`: just a regular comment will do:
```matlab
% retrieves data of a line in the visible coordinate frame

% other comments (note the empty line in-between!)
```

That way, if one calls `help cleanfigure>getVisibibleData`, you get some info on the function, and not a random comment`.